### PR TITLE
Feat/G0-5153:Advance receipts adjusted voucher list updation implemented

### PR DESCRIPTION
**Pull request by rohit-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Advance receipts for all voucher update implemented for voucher type receipts and sales
Get all advance receipts  which is adjusted by sales invoice implemented
Update all advance receipts which is adjusted from ledger

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html [hunk 1]
```diff
@@ -722,38 +722,100 @@ <h3 *ngIf="!openDropDown" (dblclick)="openHeaderDropDown()" class="d-inline-bloc
...
+                            {{ vm.baseCurrencyDetails?.symbol}}{{totalAdjustedAmount | giddhCurrency}}
+                        </p>
+                        <div class="clearfix adjustment-field">
+                            <div class="row"
+                                 *ngFor="let item of vm.selectedLedger?.advanceReceiptAdjustment?.adjustments; let entryIdx = index;">
+                                <div class="col-md-4 col-sm-5 pr-05">
+                                    <div class="form-group">
+                                        <label>Adjustment Voucher<span class="text-danger">*</span>
+                                        </label>
+                                        <input type="text" class="form-control mb-1" [(ngModel)]="item.voucherNumber" disabled>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html:790]: Add name here

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html [hunk 2]
```diff
@@ -722,38 +722,100 @@ <h3 *ngIf="!openDropDown" (dblclick)="openHeaderDropDown()" class="d-inline-bloc
...
+                                    </div>
+                                </div>
+                                <div class="col-md-4 col-sm-5 pl-05 pr-05">
+                                    <div class="form-group">
+                                        <label>Amount<span class="text-danger">*</span> <span class="text-light font-12 ml-05">(Inclusive
+                            Tax)</span></label>
+                                        <input type="text" class="form-control mb-1" *ngIf="!selectedAdvanceReceiptAdjustInvoiceEditMode"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html:797]: Add name if ngmodel is available.

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html [hunk 3]
```diff
@@ -722,38 +722,100 @@ <h3 *ngIf="!openDropDown" (dblclick)="openHeaderDropDown()" class="d-inline-bloc
...
+                                        <input type="text" class="form-control mb-1" decimalDigitsDirective [mask]="vm.inputMaskFormat"
+                                               [prefix]="vm.selectedPrefixForCurrency" (blur)="adjustedReceiptsAmountChange()"
+                                               *ngIf="selectedAdvanceReceiptAdjustInvoiceEditMode"
+                                               [(ngModel)]="item.dueAmount.amountForAccount">
+                                    </div>
+                                </div>
+                                <div class="col-sm-2 pl-05">
+                                    <div class="form-group">
+                                        <label>Tax</label>
+                                        <input type="text" class="form-control mb-1" decimalDigitsDirective [mask]="vm.inputMaskFormat"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html:809]: Add name if ngmodel is available.

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.ts [hunk 4]
```diff
@@ -1395,6 +1406,42 @@ export class UpdateLedgerEntryPanelComponent implements OnInit, AfterViewInit, O
...
+    public calculateInclusiveTaxesForAdvanceReceiptsInvoices(): void {
+        this.vm.selectedLedger.invoiceAdvanceReceiptAdjustment.adjustedInvoices.map(item => {
+            item.taxAmount = this.generalService.calculateInclusiveOrExclusiveTaxes(true, item.adjustedAmount.amountForAccount, item.taxRate, 0);
+        });
+    }
+
+    /**
+    * To calculate total amount of adjusted receipts.
+    *
+    * @param {*} event Change value of an receipts
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.ts:1427]: Update JS doc as the method doesn't take any parameter

**Pull request merged**